simple_imports/sample/import_config.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Can the sample import be run and verified periodically? We'll be updating the docker image from time to time so this will be a key regression test.

Reply:
It is set to run daily. Will need to add monitoring to detect regression failures.

Reply:
Nice! Where is this currently configured to run daily? Could we add a comment there?

Another option is to add a `dummyComments` fields to this `import_config.json` itself.

```json
{
  "dummyComments": [
    "These are explanatory comments only. They have no effect on the import.",
    "Daily sample run job: https://...",
    "TODO: Add monitoring to detect regression failures."
  ]
}
```

Reply:
Added a comment. The cronSchedule `0 20 * * *` sets this up for a daily run after this is merged.